assets/js/modules/search-console/util/index.js

```diff
@@ -110,14 +108,12 @@ export const isDataZeroSearchConsole = ( data ) => {
 		return true;
 	}
 
-	const processedData = extractSearchConsoleDashboardData( data );
-
 	const {
 		totalClicks,
 		totalImpressions,
 		averageCTR,
 		averagePosition,
-	} = processedData;
+	} = reduceSearchConsoleData( data );
```

Comment:
This might not be an obvious change:
It used to use `extractSearchConsoleDashboardData` here before but now uses the simpler `reduceSearchConsoleData` function. The important difference here is that the values here are now based on the entire report, whereas before these were based on `latestData` which was incorrect in some cases as well since we always sliced the compare range first and then latest.

https://github.com/google/site-kit-wp/blob/f6f02c5e3443611ac27a9c4b4be7dda7a03f040d/assets/js/modules/search-console/util/index.js#L89-L92

Zero data state should always be based on the entire report, not just the latest. Even though this utility is only for legacy components, it's still used so this should be an improvement/fix nonetheless.